app/views/api/integrations/apicast/shared/_mapping_rule.html.erb

```diff
@@ -13,7 +13,7 @@
       <%= p.select 'http_method', ProxyRule::ALLOWED_HTTP_METHODS, {}, html_options %>
     </td>
     <td class="pattern">
-      <%= p.text_field_with_errors :pattern, html_options %>
+      <%= p.text_field_with_errors :pattern, html_options.merge(:title => '"Catch all" mapping rules often lead to double counts when more specific rules are added.') %>
```

Comment:
I think it is better to have this text [in the locales](https://github.com/3scale/porta/blob/master/config/locales/en.yml) and here fetch it from there.

Reply:
Yes indeed.